.github/actions/composite/setupGitForOSBotify/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Setup Git for OSBotify'
+description: 'Setup Git for OSBotify'
+
+inputs:
+  GPG_PASSPHRASE:
+    description: 'Passphrase used to decrypt GPG key'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Decrypt OSBotify GPG key
+      run: cd .github/workflows && gpg --quiet --batch --yes --decrypt --passphrase=${{ inputs.GPG_PASSPHRASE }} --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
+      shell: bash
```

Comment:
NAB: move this about `run:` for consistency

Reply:
I hate how we have to pass `shell` _every time_. There's an [open PR](https://github.com/actions/runner/pull/1803) to add a default shell in composite actions, I commented in there hoping it might get pushed forward 🙂